### DISCUSSION TITLE
Central greenflag button for enhanced fullscreen addon

### DISCUSSION THREAD

**Discussion by sfederici** [ScratchAddons/ScratchAddons#7003] [Ideas]
The enhanced fullscreen addon is almost exactly what I need: I can even hide all buttons, so the fullscreen is really full. Only drawback: not having a greenflag button I have first to start the project then move to fullscreen, potentially loosing something happening in the meantime. It would be interesting to add the possibility to show a "central greenflag button" (basically, the one shown by the Scratch project player when a project page is first loaded) so that you I start the project after having enabled fullscreen, but without having visible buttons.

**Comment by DNin01** [comment]
#6034 would also solve this.

**Reply by sfederici** [reply]
When will this be available?

**Reply by DNin01** [reply]
It's currently planned for review before version 1.36.

**Reply by WorldLanguages** [reply]
Yeah, looks like that would at least help with your usecase.
Should be available in January.
